mount.go: undefined: fuse.AllowRoot ???

Trying to compile on debian 9.9

I installed golang
`go version go1.13.6 linux/amd64`

I ran go get -u github.com/dweidenfeld/plexdrive
Which outputs
```
# github.com/dweidenfeld/plexdrive/mount
gocode/src/github.com/dweidenfeld/plexdrive/mount/mount.go:52:30: undefined: fuse.AllowRoot

```
I have no idea what is wrong. I haven't been able to find anything relevant searching the web.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,6 +1,5 @@
 language: go
 
 go:
-  - 1.9
-  - 1.12
+  - 1.13
   - stable

--- a/mount/mount.go
+++ b/mount/mount.go
@@ -49,7 +49,7 @@ func Mount(
 		if "allow_other" == option {
 			options = append(options, fuse.AllowOther())
 		} else if "allow_root" == option {
-			options = append(options, fuse.AllowRoot())
+			return fmt.Errorf("The allow_root mount option is no longer supported")
 		} else if "allow_dev" == option {
 			options = append(options, fuse.AllowDev())
 		} else if "allow_non_empty_mount" == option {

